src/Tools/SemanticSearch/ReferenceAssemblies/Apis/Microsoft.CodeAnalysis.txt

```diff
@@ -542,6 +542,7 @@ Microsoft.CodeAnalysis.Diagnostics.AnalyzerFileReference.GetGenerators
 Microsoft.CodeAnalysis.Diagnostics.AnalyzerFileReference.GetGenerators(System.String)
 Microsoft.CodeAnalysis.Diagnostics.AnalyzerFileReference.GetGeneratorsForAllLanguages
 Microsoft.CodeAnalysis.Diagnostics.AnalyzerFileReference.GetHashCode
+Microsoft.CodeAnalysis.Diagnostics.AnalyzerFileReference.ToString
```

Comment:
Any idea what's up with this? I have this local change too.

Reply:
ToString was added with the [AnalyzerAssemblyRedirector API](https://github.com/dotnet/roslyn/pull/74820/files#diff-f1936ff1b73bdc55b7fc2a98dbad83e5e9aef979ec4e8ab4f0fefe8606588fb3R20). Tomas has[ a PR ](https://github.com/dotnet/roslyn/pull/77196/files#diff-519fa4fbd4d3726e24ea815b196460012a907721c851aaaf6fad590bd45c7aa4R545)to fix this up.